driver/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java

```diff
@@ -539,9 +539,9 @@ Object consume(FuzzedDataProvider data, Type genericType, AutofuzzCodegenVisitor
                                     .ignoreClassVisibility()
                                     .ignoreMethodVisibility()
                                     .enableInterClassDependencies()
-                                    .rejectPackages("jaz.*");
+                                    .rejectClasses("jaz.*");
```

Comment:
From the docs it's not clear to me if rejecting the package without the glob pattern (`*`) would work and if rejecting classes with glob pattern would also reject classes in subpackages. 
If possible, I would stick to using packages as that feels more appropriate.

Reply:
Good point. Since packages seem to be rejected together with their subpackages (https://github.com/classgraph/classgraph/blob/874442f4a1c76b2bd63284988d20c8776e0b0310/src/main/java/io/github/classgraph/ClassGraph.java#L865), I am now using this approach without any glob wildcards.